ansible/configs/ocp-workloads/examples/cakephp-openshift-applier.yml

```diff
@@ -0,0 +1,38 @@
+---
+cloud_provider: none
+env_type: ocp-workloads
+
+ocp_workloads:
+  - ocp-workload-openshift-applier
+
+ocp_username: gucore-redhat.com
+guid: testgucore
+
+openshift_cluster_content:
+
+  - object: projectrequest
+    content:
+    - name: Create namespace
+      template: https://raw.githubusercontent.com/redhat-cop/openshift-applier/v2.1.1/tests/files/templates/projectrequest.yml
+      params_from_vars:
+        NAMESPACE: "{{ guid }}"
+        NAMESPACE_DESCRIPTION: Sample namespace for student {{ ocp_username }}
+        NAMESPACE_DISPLAY_NAME: Sample namespace for student {{ ocp_username }}
+      action: "{{ applier_action }}"
+
+  - object: cakephprequest
+    content:
+    - name: Create cake-php mysql
+      template: https://raw.githubusercontent.com/openshift/origin/v4.1.0/examples/quickstarts/cakephp-mysql.json
+      params_from_vars:
+        NAME: "cakephp-mysql-{{ guid }}"
+      namespace: "{{ guid }}"
+      action: "{{ applier_action }}"
+
+target_host:
+  ansible_host: bastion.dev.openshift.opentlc.com
+  ansible_port: 22
+  ansible_user: ec2-user
+  #ansible_ssh_private_key_content: "{{ target_host_ssh_private_key_content }}"
+  ansible_ssh_private_key_file: ~/.ssh/opentlc_admin_backdoor.pem
+  #ansible_ssh_extra_args: 
```

Comment:
Minor point, but the list indenting is inconsistent in this file. I prefer not adding the extra indent before lists, but at least it should be consistent.

Reply:
ok, i just pushed a change. I prefer using extra-indent. It's also the default in yamllint IIRC.